### PR TITLE
[7.52.x][JBPM-9764] Test is missing for fix in RHPAM-3712 (empty user when by…

### PR DESCRIPTION
**Pull request by afalhambra**
**[JBPM-9764](https://issues.redhat.com/browse/JBPM-9764)**: Test is missing for fix in RHPAM-3712 (empty user when bypass is set)

### CODE REVIEW

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ByPassUserTaskServiceIntegrationTest.java [hunk 1]
```diff
@@ -83,15 +80,48 @@ public void testProcessWithUserTasks() {
...
+            // Get tasks as potential owner by using the authenticated user
+            taskList = taskClient.findTasksAssignedAsPotentialOwner("", 0, 10);
+            assertTaskList(taskList, "First task", 1);
+
+            changeUser(USER_JOHN);
+            taskList = taskClient.findTasksAssignedAsPotentialOwner("", 0, 10);
+            assertTaskList(taskList, "First task", 0);
+
+            taskList = taskClient.findTasksAssignedAsPotentialOwner(USER_MARY, 0, 10);
+            assertTaskList(taskList, "First task", 0);
```
Comment (gmunozfe) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ByPassUserTaskServiceIntegrationTest.java:107]: Same here

**Review by gmunozfe — APPROVED** [review]
Looks good to me, great job @afalhambra !
Just a minor comment in case you want to do it.

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ByPassUserTaskServiceIntegrationTest.java [hunk 2]
```diff
@@ -83,15 +80,48 @@ public void testProcessWithUserTasks() {
...
+            List<TaskSummary> taskList = taskClient.findTasksAssignedAsPotentialOwner(USER_YODA, 0, 10);
+            assertTaskList(taskList, "First task", 1);
+
+            // Get tasks as potential owner by using the authenticated user
+            taskList = taskClient.findTasksAssignedAsPotentialOwner("", 0, 10);
+            assertTaskList(taskList, "First task", 1);
+
+            changeUser(USER_JOHN);
+            taskList = taskClient.findTasksAssignedAsPotentialOwner("", 0, 10);
+            assertTaskList(taskList, "First task", 0);
```
[the remark this answers is not in the document]
Comment (afalhambra) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ByPassUserTaskServiceIntegrationTest.java]: indeed - done!

**Comment by afalhambra** [comment]
Jenkins retest this

**Comment by afalhambra** [comment]
@mareknovotny please merge along with master PR https://github.com/kiegroup/droolsjbpm-integration/pull/2520 thanks

**Pull request merged**